recipes/xerces-c/all/conanfile.py

```diff
@@ -5,15 +5,21 @@
 
 class XercesCConan(ConanFile):
     name = "xerces-c"
-    description = "Xerces-C++ is a validating XML parser written in a portable subset of C++"
+    description = (
+        "Xerces-C++ is a validating XML parser written in a portable subset of C++"
+    )
     topics = ("conan", "xerces", "XML", "validation", "DOM", "SAX", "SAX2")
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "http://xerces.apache.org/xerces-c/index.html"
     license = "Apache-2.0"
     exports_sources = ["CMakeLists.txt"]
     generators = "cmake"
     settings = "os", "arch", "compiler", "build_type"
-    options = {"shared": [True, False], "fPIC": [True, False], "char_type": ["uint16_t", "wchar_t"]}
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "char_type": ["uint16_t", "char16_t", "wchar_t"],
```

Comment:
seems like these 3 are the only supported values right now:
https://github.com/apache/xerces-c/commit/29fbbacb469752846b54bd77c7c6fec002167cbd#diff-e7cc5c8fd4dc2e5c3462f6ac46bbaec76c4c88ebc104731dc51dca0a6297837aR149